src/apps/defncopy.c

```diff
@@ -980,6 +980,10 @@ get_login(int argc, char *argv[], OPTIONS *options)
 			break;
 		case 'P':
 			password = tds_getpassarg(optarg);
+			if (!password) {
+				fprintf(stderr, "tds_getpassarg() failed\n");
```

Comment:
I would use a more user friendly error like `"Error getting password\n"`. Can I change it ?

Reply:
fixed